lib/src/main/kotlin/com/lemonappdev/konsist/api/ext/list/KoRepresentsTypeProviderListExt.kt

```diff
@@ -33,15 +33,12 @@ fun <T : KoRepresentsTypeProvider> List<T>.withoutRepresentedType(name: String,
  * @param kClasses The Kotlin classes representing the types to include.
  * @return A list containing declarations with types matching the specified Kotlin classes.
  */
-fun <T : KoRepresentsTypeProvider> List<T>.withRepresentedTypeOf(kClass: KClass<*>, vararg kClasses: KClass<*>): List<T> =
+fun <T : KoRepresentsTypeProvider> List<T>.withRepresentedTypeOf(kClass: KClass<*>?, vararg kClasses: KClass<*>?): List<T> =
```

Comment:
II'm not sure if we allow nullable KClasses. @igorwojda what do you think?

Reply:
We should not allow this. I have merged this PR and added a followup PR. https://github.com/LemonAppDev/konsist/pull/732/files